### PR TITLE
Update `util.booleanify()` return type

### PR DESCRIPTION
**Pull request by arthurvr**
Ref https://github.com/babel/babel/commit/62f37c1e6212f14a2c23d7db34a985d2a321ee2b

### CODE REVIEW

File: src/babel/util.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ export function arrayify(val: any, mapFn?: Function): Array {
   return [val];
 }
 
-export function booleanify(val: any): boolean {
+export function booleanify(val: any): any {
```
Comment (sebmck) [inline src/babel/util.js]: It should actually be removed since functions that don't have a return type are implicitly `any`.

Comment (arthurvr) [inline src/babel/util.js]: I personally like to be explicit, but of course, I'll fix it.

Comment (arthurvr) [inline src/babel/util.js]: done

**Comment by arthurvr** [comment]
Is another PR that adds tests for this function welcome? Or would those be moot?

**Comment by sebmck** [comment]
Sorry for taking so long to merge. Thank you!

**Comment by arthurvr** [comment]
You never answered this question:

> Is another PR that adds tests for this function welcome? Or would those be moot?

It just surprised me that this method was able to change its behaviour in a significant way while not any test complained.

**Comment by sebmck** [comment]
Yeah, feel free to submit tests. Test coverage isn’t 100% (it’s like ~92% or something like that) so it’s unsurprising that this isn’t directly tested.

**Comment by arthurvr** [comment]
Sure! => https://github.com/babel/babel/pull/1772

**Pull request merged**